adafruit_circuitplayground/circuit_playground_base.py

```diff
@@ -162,14 +162,24 @@ def detect_taps(self):
     @detect_taps.setter
     def detect_taps(self, value):
         self._detect_taps = value
-        if value == 1:
-            self._lis3dh.set_tap(
-                value, 90, time_limit=4, time_latency=50, time_window=255
-            )
-        if value == 2:
-            self._lis3dh.set_tap(
-                value, 60, time_limit=10, time_latency=50, time_window=255
-            )
+        if hasattr(board, "A0"):  # If we're on a CPX or CPC
```

Comment:
This is not a good solution long term because we may add another Circuit Playground that has A0 on it. #93 has some code to determine which board is which, but it obviously has not been implemented. Perhaps you can utilise part of it here. Otherwise, consider the way we did audio, it may be applicable here (or maybe not, it's simply a way we've handled differences in things, I haven't specifically looked into it).